near-sdk-sim/src/user.rs

```diff
@@ -319,7 +319,7 @@ pub fn init_simulator(genesis_config: Option<GenesisConfig>) -> UserAccount {
 ///   contract_id: "contract",
 ///   bytes: &TOKEN_WASM_BYTES,
 ///   signer_account: master_account,
-///   init_method: new(master_account_id, initial_balance.into(), "".to_string(), vec![1;32].into())
+///   init_method: new_default_meta(master_account_id, initial_balance.into())
```

Comment:
Although I learned that doc tests compile individually, it's still great to see them here ensuring that the examples stay up to date.